### ISSUE TITLE
[Feature Enhancement]: Add accessibility for SBBSlideToToggle

### ISSUE THREAD

**Issue by smallTrogdor** [SchweizerischeBundesbahnen/design_system_flutter#714]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/SchweizerischeBundesbahnen/design_system_flutter/blob/main/Contributing.md) for this project.
- [x] I have searched the [issue tracker](https://github.com/orgs/...) for an issue that matches the one I want to file, without success.

### Request type

Request for enhancement of a component

### Functionality

The SBBSlideToToggle should be screen reader accessible (the `labelText`)
